src/window.rs

```diff
@@ -175,6 +175,13 @@ impl Window {
             window.set_position(logical);
         }
 
+        // TODO: once `winit` supports getting the active monitor before window creation
+        // use with `window.with_fullscreen()` to create the window on the right monitor.
```

Comment:
I'm not sure if this is actually planned due to the technical limitations of some platforms.

So I'd recommend just removing this comment and accepting this limitation as part of the design.